common-html/footer.html

```diff
@@ -0,0 +1,5 @@
+<footer>
+  <div class="global-footer">
+    <p><img alt="" /></p>
+    <p class=>All content Copyright ニッカオフィス / knicka office</p>
```

Comment:
コピーライトは、small要素で囲みます。
```suggestion
    <p class=><small>All content Copyright ニッカオフィス / knicka office</small></p>
```

Reply:
対応しました！